website/borrel/selectors.py

```diff
@@ -0,0 +1,11 @@
+from borrel import models
+
+
+def available_products():
+    """Get products that are available for order."""
+    return models.Product.objects.filter(active=True)
+
+
+def product_categories():
+    """Get all categories."""
+    return models.ProductCategory.objects.all()
```

Comment:
What is this? It seems like you're trying to create your own ProductManager and CategoryManager (which is nice, but just build it as such then)